frontend/config/environment.js

```diff
@@ -7,6 +7,7 @@ module.exports = function(environment) {
     rootURL: "/",
     locationType: "auto",
     sentryDsn: "",
+    appVersion: undefined,
```

Comment:
Undefined for Sure? What about a fallback like V1.0?

Reply:
hmm I see your point... With undefined, it just renders nothing. The link just shows
"Version"
instead of
"Version 3.2"
Think that makes more sense than showing a wrong version number :thinking: